library/connect-go/v0.1.1/Dockerfile

```diff
@@ -2,14 +2,12 @@
 FROM golang:1.19.0-bullseye AS build
 ARG PLUGIN_VERSION
 RUN test -n "${PLUGIN_VERSION}"
-WORKDIR /app
-ENV GOBIN=/app
 RUN --mount=type=cache,target=/go/pkg/mod \
     CGO_ENABLED=0 \
     go install -ldflags="-s -w" -trimpath github.com/bufbuild/connect-go/cmd/protoc-gen-connect-go@${PLUGIN_VERSION}
 
 FROM scratch
-COPY --from=build --chown=root:root /etc/passwd /etc/passwd
-COPY --from=build /app/protoc-gen-connect-go /
+COPY --from=build --link --chown=root:root /etc/passwd /etc/passwd
```

Comment:
Didn't even know this was a command. Had to look it up.

https://docs.docker.com/engine/reference/builder/#benefits-of-using---link